addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
Either the tag "backwards-compatibility" means something or not. I am fine if the majority says that they don't care but I won't maintain a version that only pretends something. So the options are:

1) bump backwards-compatibility
2) drop backwards-compatibility

I won't do crap in between. Tags have to be reliable.